splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -492,18 +510,34 @@ def generate(source, config, ta_version, outputdir=None, python_binary_name="pyt
         )
 
         logger.info("Copy globalConfig to output")
-        shutil.copyfile(
-            config,
-            os.path.join(
-                outputdir,
-                ta_name,
-                "appserver",
-                "static",
-                "js",
-                "build",
-                "globalConfig.json",
-            ),
-        )
+        
+        if is_global_config_yaml:
+            shutil.copyfile(
+                config,
+                os.path.join(
+                    outputdir,
+                    ta_name,
+                    "appserver",
+                    "static",
+                    "js",
+                    "build",
+                    "globalConfig.yaml",
+                ),
+            )
+        else:
+            shutil.copyfile(
+                config,
+                os.path.join(
+                    outputdir,
+                    ta_name,
+                    "appserver",
+                    "static",
+                    "js",
+                    "build",
+                    "globalConfig.json",
+                ),
+            )
```

Comment:
I think it will be easier to have 1 shutil.copyfile and use if else inside the os.path.join statement.

Reply:
Done